code/modules/cargo/expressconsole.dm

```diff
@@ -30,6 +29,8 @@
 		to_chat(user, "<span class='notice'>You insert the disk into [src], allowing for advanced supply delivery vehicles.</span>")
 		qdel(W)
 		return TRUE
+	else

```

Comment:
you may just want to remove the else and indentation for ..() as in the past it's caused some **unique** interactions

Reply:
what kinda interactions? also done

Reply:
I've semi-broken signallers because of doing just that.